.eleventy.js

```diff
@@ -140,33 +154,59 @@ module.exports = function (config) {
   let markdownLibrary = markdownIt({
     html: true,
     breaks: true,
-    linkify: true
-  }).use(markdownItAnchor, {
-    permalink: markdownItAnchor.permalink.ariaHidden({
-      placement: 'after',
-      class: 'direct-link',
-      symbol: '',
-      level: [1, 2, 3, 4],
-    }),
-    slugify: config.getFilter('slug'),
-  }).use(markdownItTable);
+    linkify: true,
+    typographer: true
+  })
+    .use(function (md) {
+      // Override the default link rendering rule to add target and rel attributes
+      const defaultRender = md.renderer.rules.link_open || function (tokens, idx, options, env, self) {
+        return self.renderToken(tokens, idx, options);
+      };
+      md.renderer.rules.link_open = function (tokens, idx, options, env, self) {
+        const token = tokens[idx];
+        const hrefAttr = token.attrs ? token.attrs.find(attr => attr[0] === 'href') : null;
+        if (hrefAttr && /^https?:\/\//.test(hrefAttr[1])) {
+          token.attrSet('target', '_blank');
```

Comment:
this is awesome and makes me happy. If you also add a class ("usa-link--external") I think we get the little visual indicator [docs](https://designsystem.digital.gov/components/link/#component-variants-link) which would be nice in the article content.